app-connector-client/server/config.ts

```diff
@@ -0,0 +1,30 @@
+interface varkesConfigInterface {

```

Comment:
that are two different ways of configuration in one file, maybe separate it?
Why one is defined as interface, the other not?
Why not just changing the varkesConfig reading into a regular JSON file parsing instead of using the "require" method requiring such strange interface?

Reply:
changing varkesConfig to JSON is better